python/tvm/contrib/hexagon/session.py

```diff
@@ -248,8 +248,8 @@ def get_aot_executor(
         GraphModule :
             Runtime graph module that can be used to execute the graph.
         """
-        aot_mod = self.load_module(module_file)
-        return tvm.runtime.executor.AotModule(aot_mod["default"](self.device))
```

Comment:
Please add a github issue describing the problem and link it here. Also mention that this is a temporary fix

Reply:
Filed https://github.com/apache/tvm/issues/13741